docs/HowTo/Find-and-Connect/Specifying-NAT.md

```diff
@@ -0,0 +1,65 @@
+description: Configuring NAT with Hyperledger Besu
+<!--- END of page meta data -->
+
+# Configuring NAT 
+
+Use the [`--nat-method`](../../Reference/CLI/CLI-Syntax.md#nat-method) option to specify the 
+NAT method. Options are: `UPNP`, `MANUAL`, `DOCKER`, `AUTO`, and `NONE`. 
+
+The [enode](../../Concepts/Node-Keys.md#enode-url) advertised to other nodes during discovery is the 
+external IP address and port. The [`admin_nodeInfo`](../../Reference/API-Methods.md#admin_nodeinfo) 
+JSON-RPC API method returns the external address and port for the `enode` and `listenAddr` properties. 
+  
+While Hyperledger Besu is running, the following are not supported: 
+
+* IP address changes
+* Changing NAT methods. To change NAT method, restart the node with the [`--nat-method`](../../Reference/CLI/CLI-Syntax.md#nat-method)
+option set as required. 
+
+# UPnP
```

Comment:
Heading level 2 instead of 1?

Reply:
Yes - good catch.  I was looking at the rendered version yesterday wondering why there was so much white space after the headings.